ansible/templates/etc/systemd/system/stickypedia.service.j2

```diff
@@ -0,0 +1,22 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description=Run stickypedia (gollum)
+After=network.target
+OnFailure=failure-notificator@%n.service
+
+[Service]
+Type=simple
+User=stickypedia
+Group=www-data
+Restart=always
+
+WorkingDirectory=/var/www/stickypedia/stickypedia.{{ canonical_hostname }}
+
+RuntimeDirectory=wiki
+RuntimeDirectoryMode=0770
+
+ExecStart=gollum
```

Comment:
This won't work, systemd only allows you to start executables using the fully qualified path to it. you'll need to find out where `gem` installs this

Reply:
also done